Unity/Assets/Ermfish/Databank/Databank.asset

```diff
@@ -17,7 +17,8 @@ MonoBehaviour:
   isImportantUnlock: 0
   scanSounds: {fileID: 8990697991671373227, guid: 1138c2d796e81ac43a4f257daf9651b7,
     type: 2}
-  encyPath: Lifeforms/Fauna/SmallHerbivores
+  encyPathSN: Lifeforms/Fauna/SmallHerbivores
+  encyPathBZ: Research/Lifeforms/Fauna/SmallHerbivores
```

Comment:
Is it really supposed to start with Research?

Reply:
Yeah, in BZ they put a bunch of the entries under Research
![image](https://github.com/Alexejhero/Neurosama-Subnautica-Mod/assets/3830522/63d9b9c2-c1c7-41d0-a13e-e19e21944cb1)

The `Lifeforms/whatever` are still in the language files but are actually unused (on my endgame save all the entries are under Research)